src/redux/actions/types.js

```diff
@@ -19,6 +19,7 @@ export const FETCH_DICTIONARY_CONCEPT = '[concepts] fetch concepts of one dictio
 export const CLEAR_DICTIONARIES = '[dictionaries] clear dictionaries';
 export const FETCHING_DICTIONARY = '[dictionary] fetch dictionary';
 export const CLEAR_DICTIONARY = '[dictionary] clear dictionary';
+export const REMOVE_CONCEPT = '[concepts] remove concept';
```

Comment:
I would recommend using action verbs for the action types. For example in this case `REMOVING_CONCEPTS`.

Reply:
That is a very good recommendation using a verb, but since we will be removing only a single concept I don't think changing to REMOVING will add any significant change.